jobs/dcp1-version/README.md

```diff
@@ -0,0 +1,44 @@
+# Populating firstDcpVersion and dcpVersion of DCP1 metadata
+
+The script `populate_dcp_version.py` fixes the missing `dcpVersion` values in the metadata documents in the Ingest database. It reads `dcp1-metadata.txt.tar.gz`  which is the compressed file that contains the list of all the metadata json files in the DCP1 terra bucket `gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/no-analysis/metadata`.
+
+Please see issue: https://github.com/ebi-ait/dcp-ingest-central/issues/481
+
+## How to build docker image and push to quay.io?
+
+```
+$ docker build . -t quay.io/ebi-ait/ingest-kube-deployment:dcp1-version_20211102.1
+$ docker push quay.io/ebi-ait/ingest-kube-deployment:dcp1-version_20211102.1
+```
+
+## How to run inside ingest cluster?
+Source environment config and set correct environment context 
+```
+$ source config/environment-dev
+$ kubectx ingest-eks-dev
+```
```

Comment:
I think the `$` make it more difficult to copy and past the code examples.
```suggestion
```bash
source config/environment-dev
kubectx ingest-eks-dev
```
```